hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CacheCreateConfigMessageTask.java

```diff
@@ -90,7 +90,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "createConfig";
```

Comment:
is there a user method as createConfig for ICache? I think this should be createCacheConfigOnAllMembers